pkgs/development/libraries/blkhash/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchFromGitLab
+, makeWrapper
+, meson
+, ninja
+, pkg-config
+, cmake
+, openssl
+, enableLibnbd ? stdenv.isLinux, libnbd
```

Comment:
isLinux is likely correct now, but I think a better way woul be to add the
dependency dynanically if a libnbd nix package is available for the target.
But I think it is good enough for now.